manifest.yaml

```diff
@@ -0,0 +1,20 @@
+---
+applications:
+- name: keycloak
+  routes:
+    - route: identity-idva-keycloak-((ENVIRONMENT_NAME)).apps.internal
+  instances: 1
+  health-check-type: process
+  memory: 1.2G
+  disk_quota: 2G
+  command: ./install.sh && keycloak-$(cat KEYCLOAK_VERSION)/bin/kc.sh start --auto-build
```

Comment:
Can we move `./install.sh` to the GitHub action to support having this in a restricted or closed space-type?

Reply:
Could. This will require downloading files twice which is annoying. Maybe what we should do is create a custom buildpack. That runs with the staging security group i think which would let it download files and then run with the restricted security group

Reply:
@dzaslavskiy I don't understand -- why would files be downloaded twice? Once the gha installs them, they'll just be included in the deployment upload right? The the command would just be `kc.sh start --auto-build`?

Reply:
download once to github, upload from github to cg

Reply:
The default environment `dev|test|prod` will soon have a restricted egress policy attached to it. When that policy is applied, it will break this install process unless the GHA performs that install. I think it's best to move it to GHA to avoid any potential deployment failures once we start enforcing egress policies.